Jropp/jason-ropp-js-iobp
# Onboarding Project: Card Expand and Collapse

## What It Does

Adds expansion and collapse of new user card when "Create New User" button is clicked. Also adds a few other minor styling changes including color scheme, and the display of the users ID.

This is the first round of significant style changes. 

I added `@keyframes` that adjust the height and width of the new user card as well as the opacity of the input items contained in that user card.

Closes #62 
Closes #63 

## How To Test

Click the "Create New User" button, then click cancel. There should be a pleasing visual woosh as the card expands and collapses. Somehow you know that because of this, the rest of your day will go splendidly. 

## Checklist

Under penalty of public shaming, I avow that I:


Browsers tested:

## Dependencies

- [ ] #56  
- [ ] #58
